src/main/java/com/synopsys/integration/alert/web/api/certificate/CertificateActions.java

```diff
@@ -69,6 +70,43 @@ public CertificateActions(CustomCertificateAccessor certificateAccessor, Certifi
                    .map(this::convertFromDatabaseModel);
     }
 
+    public MessageResult validateCertificate(CertificateModel certificateModel) {
+        CustomCertificateModel convertedModel = convertToDatabaseModel(certificateModel);
```

Comment:
The class names are confusing to me here. We have a CertificateModel that we run convertToDatabaseModel which changes it to a CustomCertificateModel. I feel like the name CustomCertificateModel is misleading? Or perhaps the convertToDatabaseModel Should be changed?

Reply:
The database API serves a `CustomCertificateModel`. The REST API serves a `CertificateModel`.

Reply:
I guess I'm confused about what a custom certificate is. Is it just a self signed certificate? Or is the user somehow customizing this model?

Reply:
It would probably be better to call it a `UserProvidedCertificate` or something, but this code is actually just moved code from lowing in the class. I could make the change in this PR, but it's slightly out of scope.